docs/topics/i18n/translation.txt

```diff
@@ -825,6 +825,10 @@ Each ``RequestContext`` has access to three translation-specific variables:
   right-to-left language, e.g.: Hebrew, Arabic. If False it's a
   left-to-right language, e.g.: English, French, German etc.
 
+.. versionchanged:: 1.8
+
+  The `i18n` context processor is not enabled by default for new projects.
```

Comment:
Actually I'm a little unsure whether or not to use backticks at all. I'm a little inconsistent. What do you think?